.vscode/launch.json

```diff
@@ -0,0 +1,35 @@
+{
```

Comment:
Do we need vscode in the project? I think that we need to add it to gitignore

Reply:
removed and added to .gitignore